client/components/jetpack/licensing-activation/index.tsx

```diff
@@ -22,6 +24,7 @@ interface Props {
 }
 
 const LicensingActivation: FC< Props > = ( {
+	altLayout,
```

Comment:
Instead of `altLayout`, can we directly use feature flag here?

Reply:
Hi @manzoorwanijk, is it okay to change the layout for the old flow? I actually would prefer your suggestion if we do not need to retain the old layout for non-standalone instructions. I can confirm this w/ the design team first on how they feel about it.

Reply:
I think we are going to retain the existing behavior for classic Jetpack plugin.

Reply:
then I guess we retain the `altLayout`. or maybe you can suggest better name if you think this is not a good one.

Reply:
I don't have a better name in my mind 😄